apps/demo-wallet-sdk-react/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@fuel-wallet/react": "0.15.2",
-    "fuels": "0.73.0",
+    "fuels": "0.76.0",
```

Comment:
Should it be using it from the `workspace:*`?

Reply:
NIce catch 👍